core/src/main/scala/com/rallyhealth/vapors/factfilter/dsl/ExprBuilder.scala

```diff
@@ -79,6 +82,23 @@ final class SubtractBuilderOps[R : Subtraction](number: R) {
 sealed class FoldableExprBuilder[F[_] : Foldable, V, M[_] : Foldable, U, P](returnOutput: Expr[F, V, M[U], P])
   extends ExprBuilder[F, V, M, U, P](returnOutput) {
 
+  def toList(
+    implicit
+    ev: M[U] <:< Iterable[U],
+    captureOutput: CaptureP[F, V, List[U], P],
+  ): FoldableExprBuilder[F, V, List, U, P] =
+    to(List)
+
+  def to[N[+_] <: Iterable[_] : Foldable](
```

Comment:
This is quite a type. Does it need to be Iterable and Foldable?

Reply:
Good catch. I only need the `Foldable` typeclass.